.github/workflows/settings.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+    <profiles>
+        <profile>
+            <id>ossrh</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <releaseRepository-Id>ossrh</releaseRepository-Id>
+                <releaseRepository-URL>https://oss.sonatype.org/service/local/staging/deploy/maven2/
+                </releaseRepository-URL>
+                <snapshotRepository-Id>ossrh</snapshotRepository-Id>
+                <snapshotRepository-URL>https://oss.sonatype.org/content/repositories/snapshots</snapshotRepository-URL>
+                <project.scm.id>github</project.scm.id>
+                <gpg.executable>gpg</gpg.executable>
+                <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
+            </properties>
+        </profile>
+    </profiles>
+
+    <servers>
+        <server>
+            <id>ossrh</id>
+            <username>${env.SONATYPE_USERNAME}</username>
+            <password>${env.SONATYPE_PASSWORD}</password>
+        </server>
+        <server>
+            <id>github</id>
+            <username>${env.X_GITHUB_USERNAME}</username>
```

Comment:
why is the env var prefixed with `X_`, looks ugly :D or is that by convention?

Reply:
copied from aem-core-components, not sure?

Reply:
nothing blocking, just ugly xD